Formula/amazon-efs-utils.rb

```diff
@@ -180,6 +182,10 @@ def install
     inreplace buildpath/"dist/efs-utils.conf", "/var", libexec/"var"
     inreplace buildpath/"dist/amazon-efs-mount-watchdog.plist", "/var", libexec/"var"
 
+    if libexec.to_s == "/opt/homebrew/Cellar/amazon-efs-utils/#{version}/libexec"
```

Comment:
This probably is fine, while do we need the explicit validation? Or can we just check whether `/opt/homebrew` is a substring of libexec?

Reply:
We could compare prefix only. Thanks for the idea please check a reivision.